server/datastore/mysql/migrations/tables/20220708152154_IncreaseBatteryHealthLength.go

```diff
@@ -0,0 +1,20 @@
+package tables
```

Comment:
I don't think we need a migration (because the battery changes weren't released to public), so you could amend the original migration code.

Reply:
gotcha, I was thinking about us mostly, wouldn't that require all devs to reset their dbs?

Reply:
Yes, devs would need to reset their dbs.

Reply:
and/or devs would need to manually run the SQL command on their DB. The same would need to happen with Dogfood.

We should probably come up with guidelines as to when modifying migrations is acceptable vs. creating new migrations. For this case, happy to go with whatever you guys decide.

Reply:
Ah dogfood, it's on `4.16.0`. So we should be safe there (because the guideline is to not do downgrades there IIRC).

+1, we need guidelines.

I'm ok to leave this as-is as it's a new table anyways.

Reply:
went back and removed the migration 🧹